scripts/collections.cdc

```diff
@@ -124,6 +125,24 @@ pub fun main(address: Address) : {String : MetadataCollection}? {
 	}
 	*/
 
+
+	let rareRoomCollection = account.getCapability<&RareRooms_NFT.Collection{RareRooms_NFT.RareRooms_NFTCollectionPublic}>(RareRooms_NFT.CollectionPublicPath)
+	if rareRoomCollection.check() {
+		let rareRoomNfts = rareRoomCollection.borrow()!.getIDs()
+        let items: [MetadataCollectionItem]=[]
+        for id in rareRoomNfts {
+            let nft = rareRoomCollection.borrow()!.borrowRareRooms_NFT(id: id)!
+			items.append(MetadataCollectionItem(
+				id: id,
+				name: RareRooms_NFT.getSetMetadataByField(setId: nft.setId, field: "name")!,
+				// we use "preview" and not "image" because of potential .glg and .mp4 file types
+				image: RareRooms_NFT.getSetMetadataByField(setId: nft.setId, field: "preview")!,
+				url: "https://app.rarerooms.io"  // or use field "external_url"
```

Comment:
Same here, it could be this instead if you want a link to the detail page

```suggestion
				url: "https://rarerooms.io/tokens/".concat(id.toString())
```